content/docs/1.6.0/high-availability/recover-volume-when-read-only.md

```diff
@@ -0,0 +1,11 @@
+---
+  title: Recover Volume When It Becomes Read Only
+  weight: 1
+---
+
+The state of a volume can change to read-only when IO errors occur. IO errors can be caused by network disconnection, high disk latency, abnormal node shutdown, and other issues.
```

Comment:
```suggestion
The state of a volume can change to read-only when IO errors occur. IO errors can be caused by a variety of issues, including the following:

- Network disconnection: Interrupted connection between the engine and replicas
- High disk latency: Significant delay in the transfer of data between a replica and the corresponding disk
- Abnormal node shutdown: Failure of a node that is running the engine
```

Reply:
@derekbit are there any missing or incorrect changes about the causes of IO errors? :point_up:

Reply:
> Abnormal node shutdown: Failure of a node that is running the engine

When an engine is somehow shutdown, application and engine are both gone, so the IO error shouldn't happen.

> abnormal node shutdown

`abnormal node shutdown` in my previous comment might be not necessary.

WDYT? @ChanYiLin @innobead

Reply:
Yeah agree, and I think here we just want to provide some hints about why it happens
But IO errors may be caused by many factors, I don't think we can and need to  list all of them.

Reply:
Fixed.